File: sum.cxx
//A sum program
#include<iostream>
#include<cstdlib>    //double atof(char*)
using namespace std;

int main(void)
{
	double m;
	double n;
	char a[10];
	char b[10];
	cout<<"Please input the first digit:";
	cin>>a;
	m=atof(a);
	cout<<"Please input the second digit:";
	cin>>b;
	n=atof(b);
	cout<<"The results is:"<<m+n<<endl;

	
	return 0;
}

File: sum.py
#Author:ryt
#!/usr/bin/python

print("*****This is sum program,please input two digit*****")
a=input("input first digit:")
a=str(a)
while not a.isdigit():
	print("The input is not invalid!Please input a digit")
	a=input("input first digit:")
a=int(a)
b=input("input second digit:")
b=str(b)
while not b.isdigit():
	print("The input is not invalid!Please input a digit")
	b=input("input the second digit:")
b=int(b)
print("result:%f"%(a+b))
exit(0)
